src/lib/dom-api-shady.html

```diff
@@ -81,6 +82,11 @@
         } else {
           TreeApi.Composed.appendChild(container, node);
         }
+      // if a fragment provoked distribution, clean up it's actual dom.
+      } else if (isFragment) {
```

Comment:
Ah my bad, the multiple `if`s next to each other was confusing, and I assumed the `else if (isFragment)` was part of the `if (!isFragment)` flow.